SQL/tgstation_schema.sql

```diff
@@ -263,6 +263,9 @@ CREATE TABLE `poll_question` (
   `question` varchar(255) NOT NULL,
   `adminonly` tinyint(1) DEFAULT '0',
   `multiplechoiceoptions` int(2) DEFAULT NULL,
+  `createdby_ckey` varchar(45) NULL DEFAULT NULL,
+  `createdby_ip` varchar(45) NULL DEFAULT NULL,
+  `for_trialmin` varchar(45) NULL DEFAULT NULL,

```

Comment:
Just to keep it inline with the table used on our server, these columns were all added by errorage to use with his stat-tracking site and he never updated the code with them.
That said, I don't think they have any function beyond record-keeping.